### ISSUE TITLE
Creating development environment with pip & virtualenv - multiple pull requests

### ISSUE THREAD

**Issue by pranayrauni** [MDAnalysis/UserGuide#202]
This issue is raised to create development environment `with pip & virtualenv` smoothly. 

- On page: https://userguide.mdanalysis.org/stable/contributing_code.html#working-with-mdanalysis-docs

**Issues:**

- There is no bin folder/file in virtual environment directory `my-project-env` . `Scripts` folder is there. So to activate virtual environment the command should be like this `source myproject-env/Scripts/activate`

- Adding `source /usr/local/bin/virtualenvwrapper.sh` in `bashrc` or `bash_profile` gives error.

- Error while generating docs with sphinx because of older version of sphinx.

**Req:**

- `MDAnalysis as mda` version: 2.2.0-dev0
- Python version: 3.10.0
- OS: Windows

**Comment by lilyminium** [comment]
Thanks for raising this issue, @pranayrauni. I'd really rather store some environment files in MDAnalysis instead of continuing to update a morass of dependencies in the user guide. @IAlibay it should be possible to add this as a GitHub Action in CI, right?

**Comment by IAlibay** [comment]
I'm not sure I want to replace our current dependency actions with an env file at this point (since we rely on modularity for some of the tests) - if that's what you're proposing.

**Comment by lilyminium** [comment]
No, like use CI to write environment files to the repo — I’ll try to mock something up over the weekend. That way at least it stays current with whatever works with CI.

**Comment by IAlibay** [comment]
I see, yeah that's feasible, albeit possibly a bit messy.

**Comment by jandom** [comment]
@lilyminium could we close this one out? Seems inactive for about a year

**Comment by RMeli** [comment]
@jandom Thanks for looking at old issues. I also added a "close?" label as in the main repo in case you need it (but inactive is good too).

**Comment by jandom** [comment]
Great, sorry I wasn't aware – let's harmonize and call the label here 'close' as well, rather than inactive

**Comment by IAlibay** [comment]
I think this might be better to keep around for now. On the fly environment files are a doable thing, just needs a bit of thinking on how best to go about it really.
